### PR TITLE
patch-2

### LINKED ISSUE #21: Typos in script
- Line 60 (remove blank line).
- Line 10 (`: -` => `:`).

### PR DESCRIPTION
**Pull request by Oleg-Chashko**
Removed extra whitespace, fix Typos and fix #21

### CODE REVIEW

**Comment by htcfreek** [comment]
@Oleg-Chashko 
Is it possible that you fix #21 in your PR too? Thank you.

**Comment by htcfreek** [comment]
@Oleg-Chashko 
Is it possible that you fix #21 in your PR too? Thank you.

File: src/GetDiskInfoFromWmi/GetDiskInfoFromWmi.au3 [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 #cs
 ===============================================================================================================================
- Title ...............: _GetDiskInfoFromWmi (GitHub: - https://github.com/htcfreek/AutoIt-Scripts)
+ Title ...............: _GetDiskInfoFromWmi (GitHub https://github.com/htcfreek/AutoIt-Scripts)
```
Comment (htcfreek) [inline src/GetDiskInfoFromWmi/GetDiskInfoFromWmi.au3]:
```suggestion
 Title ...............: _GetDiskInfoFromWmi (GitHub: https://github.com/htcfreek/AutoIt-Scripts)
```

**Review by htcfreek — CHANGES_REQUESTED** [review]

**Comment by Oleg-Chashko** [comment]
Ready to merge?

**Comment by Oleg-Chashko** [comment]
Please check, has everything been fixed already?

**Comment by htcfreek** [comment]
Why did you closed this?

**Comment by Oleg-Chashko** [comment]
There was no merge, I deleted it when it was not needed.

**Comment by htcfreek** [comment]
I only had not the time to look on it.

**Pull request closed without merging**